site/content/3.13/data-science/arangographml/arangograph-ml.md

```diff
@@ -0,0 +1,183 @@
+---
+title: ArangoGraphML Web Interface
+menuTitle: ArangoGraphML Web Interface
+weight: 15
+description: >-
+ Enterprise-ready, graph-powered machine learning as a cloud service or self-managed
+aliases:
+  - getting-started-with-arangographml
+---
+Solve high-computational graph problems with Graph Machine Learning. Apply ML on a selected graph to predict connections, get better product recommendations, classify nodes, and perform node embeddings. Configure and run the whole machine learning flow entirely in the web interface.
+
+## Creating a GraphML Project
+
+To create a new GraphML project using the ArangoDB Web Interface, follow these steps:
+
+- **Select the Target Database** – From the **Database** dropdown in the left-hand sidebar, select the database where the project should reside.
```

Comment:
These are steps that should be followed in order, so use an ordered list here.
dropdown -> dropdown menu (or simply just write to select the database without mentioning the specific widget type)